notebook/static/notebook/js/notebook.js

```diff
@@ -771,27 +750,23 @@ define(function (require) {
                 if (this.mode !== 'command') {
                     this.command_mode();
                 }
+                this.get_cell(sindex).unselect();
             }
-            var current_selection = this.get_selected_cells();
-            for (var i=0; i<current_selection.length; i++) {
-                current_selection[i].unselect();
+            var cell = this.get_cell(index);
+            cell.select();
+            if (cell.cell_type === 'heading') {
```

Comment:
Great! Confusing, but great, and I think we should revert to how you have
it here.

On Fri, Oct 16, 2015 at 1:21 PM, Jonathan Frederic <notifications@github.com
> wrote:

> In notebook/static/notebook/js/notebook.js
> <https://github.com/jupyter/notebook/pull/612#discussion_r42266894>:
>
> >              }
> > -            var current_selection = this.get_selected_cells();
> > -            for (var i=0; i<current_selection.length; i++) {
> > -                current_selection[i].unselect();
> > +            var cell = this.get_cell(index);
> > +            cell.select();
> > +            if (cell.cell_type === 'heading') {
>
> Also code that existed before multiselection, just reverted.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/notebook/pull/612/files#r42266894>.
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com